content/hi/docs/contribute/review/for-approvers.md

```diff
@@ -0,0 +1,202 @@
+---
+title: अनुमोदकों और समीक्षकों के लिए समीक्षा करना
+linktitle: अनुमोदकों और समीक्षकों के लिए
+slug: for-approvers
+content_type: concept
+weight: 20
+---
+
+<!-- अवलोकन -->
+
+SIG Docs [समीक्षक](/docs/contribute/participate/#reviewers) और
+[अनुमोदक](/docs/contribute/participate/#approvers) परिवर्तन की समीक्षा करते समय
+कुछ अतिरिक्त चीज़ें करते हैं।
+
+प्रत्येक सप्ताह एक विशिष्ट प्रलेखन अनुमोदक स्वेच्छा से पुल अनुरोधों को ट्रीआज और समीक्षा करता है। यह व्यक्ति उस सप्ताह के लिए "PR रैंगलर (PR Wrangler)" है। 
+अधिक जानकारी के लिए [PR रैंगलर शेड्यूलर](https://github.com/kubernetes/website/wiki/PR-Wranglers) देखें। PR रैंगलर बनने के लिए, साप्ताहिक SIG Docs मीटिंग में भाग लें और स्वयंसेवक बनें। यहां तक कि अगर आप वर्तमान सप्ताह के लिए शेड्यूल पर नहीं हैं, आप  तब भी उन पुल अनुरोधों (PRs) की समीक्षा कर सकते हैं जिनकी सक्रिय रूप से समीक्षा नहीं की गई है।
```

Comment:
Volunteer noun = स्वयंसेवक. The intended meaning of the sentence is to nominate yourself voluntarily for the PR wrangler position. I feel the below suggestion captures it better, WDYT?
```suggestion
अधिक जानकारी के लिए [PR Wrangler अनुसूचक](https://github.com/kubernetes/website/wiki/PR-Wranglers) देखें। PR Wrangler बनने के लिए, साप्ताहिक SIG Docs मीटिंग में भाग लें और स्वेच्छा से खुद को नामांकित करें। यहां तक कि अगर आप वर्तमान सप्ताह के लिए शेड्यूल पर नहीं हैं, आप  तब भी उन पुल अनुरोधों (PRs) की समीक्षा कर सकते हैं जिनकी सक्रिय रूप से समीक्षा नहीं की गई है।
```